app/messaging/create-message.js

```diff
@@ -0,0 +1,10 @@
+const createMessage = (body, type, options) => {
+  return {
+    body,
+    type,
+    source: 'ffc-pay-statement-constructor',
```

Comment:
Should this be hard-coded?

Reply:
I think it should be stored in the config as constant.

Reply:
sounds sensible

Reply:
Added additional field to config and updated dependecies and tests